android/Omnivore/app/src/main/java/app/omnivore/omnivore/persistence/entities/SavedItem.kt

```diff
@@ -148,9 +161,73 @@ interface SavedItemDao {
       "ORDER BY publishDate DESC"
   )
   fun getLibraryLiveDataSortedByRecentlyPublished(archiveFilter: Int): LiveData<List<SavedItemCardDataWithLabels>>
+
+  @Transaction
+  @Query(
```

Comment:
@jacksonh Nice work here! 🙌🏽

Reply:
@satindar thanks! Does this look OK to merge in general? I think I can cleanup a bit after this, removing some of the specific data classes that we might not need anymore. 

Still get some loading flicker, learning how this LiveData stuff works.